bitcash/network/services.py

```diff
@@ -1,13 +1,13 @@
-import logging
 import os
 import requests
-from decimal import Decimal
 
-from bitcash.exceptions import InvalidNetwork
-from bitcash.network import currency_to_satoshi
-from bitcash.network.meta import Unspent
-from bitcash.network.transaction import Transaction, TxPart
+# Import supported endpoint APIs
+from bitcash.network.APIs.BitcoinDotComAPI import BitcoinDotComAPI
 
+# Dictionary of supported endpoint APIs
+ENDPOINT_ENV_VARIABLES = {"BITCOINCOM": BitcoinDotComAPI}
+
+# Default API call total time timeout
 DEFAULT_TIMEOUT = 30
```

Comment:
Why is there a different `DEFAULT_TIMEOUT` in `services.py` and `BitcoinComAPI.py`?